frontend/src/pages/applynewclient/formsections/FormSections.vue

```diff
@@ -22,15 +22,51 @@
 </template>
 
 <script setup lang="ts">
-import { computed } from "vue";
+import { computed, onMounted, ref } from "vue";
 import _ from "lodash";
 import FormSectionTemplate from "./FormSectionTemplate.vue";
 import { businessTypeSectionSchema } from "../formsectionschemas/BusinessTypeSectionSchema";
 import { informationSectionSchema } from "../formsectionschemas/InformationSectionSchema";
 import { locationSectionSchema } from "../formsectionschemas/LocationSectionSchema";
 import { formData } from "../../../store/newclientform/FormData";
+import axios from "axios";
+
+const backendUrl = import.meta.env.VITE_BACKEND_URL;
+axios.defaults.headers.common['Access-Control-Allow-Origin'] = '*';
+
+const clientTypeOptions = ref([]);
+
+onMounted(async () => {
+  const activeClientTypeCodes: FormSelectOptionType[] = [];
+  const response = await axios.get(`${backendUrl}/api/clients/activeClientTypeCodes`, {});
```

Comment:
I'm not happy with the actual format of this response, we should discuss a better schema for this later